modular_zubbers/code/modules/customization/species/proteans/organs/protean_brain.dm

```diff
@@ -159,16 +144,22 @@
 		eyes = new /obj/item/organ/eyes/robotic/protean
 		eyes.on_bodypart_insert()
 		eyes.Insert(owner, TRUE)
+	else
+		eyes.set_organ_damage(0)
 
 	if(isnull(tongue))
 		tongue = new /obj/item/organ/tongue/cybernetic/protean
 		tongue.on_bodypart_insert()
 		tongue.Insert(owner, TRUE)
+	else
+		tongue.set_organ_damage(0)
 
 	if(isnull(ears))
 		ears = new /obj/item/organ/ears/cybernetic/protean
 		ears.on_bodypart_insert()
 		ears.Insert(owner, TRUE)
+	else
+		ears.set_organ_damage(0)
```

Comment:
iffy on letting them having fullheals on these organs but i suppose these do delete on removal, any other reasoning for this? Cybernetic replacements should still work, i think?